src/components/ContextCard.vue

```diff
@@ -0,0 +1,400 @@
+                  <div class="view-description">{{view.description}}</div>
+                </div>
+                <div v-bind:key="i" class="padding"/>
+              </template>
+              <div style="margin-bottom: 16px;"/>
+              <div v-if="contextData.samples && contextData.samples.length > 0" class="subtitle">Samples on Scaffold</div>
+              <template v-for="(sample, i) in contextData.samples">
+                  <span v-bind:key="i+'_3'" class="context-card-item cursor-pointer" @click="toggleSampleDetails(i)">
+                    <div v-bind:key="i+'_6'" style="display: flex">
+                      <div v-if="sample.color" class="color-box" :style="'background-color:'+ sample.color"></div>
+                      <img class="key-image" v-else-if="sample.thumbnail" :src="getFileFromPath(sample.thumbnail)">
+                      {{sample.heading}}
+                      <i class="el-icon-warning-outline info"></i>
+                    </div>
+                  </span>
+                  <div v-bind:key="i+'_4'" v-if="sampleDetails[i]" v-html="sample.description"/>
+                  <a v-bind:key="i+'_5'" v-if="sampleDetails[i] && sample.path" :href="generateFileLink(sample)" target="_blank">View Source</a>
+                  <div v-bind:key="i+'_2'" class="padding"/>
+              </template>
+            </template>
+
+            <!-- Show samples under views if the ids match -->
+            <template v-else>
+              <div v-if="contextData.views && contextData.views.length > 0" class="subtitle">Scaffold Views</div>
+              <template v-for="(view, i) in contextData.views">
+                <span :key="i+'_1'" @click="viewClicked(view, i)" class="context-card-view">
+                  <img class="view-image" :src="getFileFromPath(view.thumbnail)"/> 
+                  <div class="view-description">{{view.description}}<i class="el-icon-warning-outline info"></i> </div>
+                </span>
+                <div v-if="sampleDetails[i]" v-html="samplesMatching(view.id).description" :key="i+'_2'"/>
+                <a v-bind:key="i+'_5'" v-if="sampleDetails[i] && samplesMatching(view.id).path" :href="generateFileLink(samplesMatching(view.id))" target="_blank">View Source</a>
+                <div :key="i" class="padding"/>
+
+                <!-- Extra padding if sample details is open -->
+                <div :key="i+'_6'" v-if="sampleDetails[i]" class="padding"/>
+              </template>
+            </template>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script>
+/* eslint-disable no-alert, no-console */
+import Vue from "vue";
+import { Link, Icon, Card, Button, Select, Input } from "element-ui";
+import lang from "element-ui/lib/locale/lang/en";
+import locale from "element-ui/lib/locale";
+import EventBus from "./EventBus"
+
+//provide the s3Bucket related methods and data.
+import S3Bucket from "../mixins/S3Bucket";
```

Comment:
This file is not included in previous commit.

Reply:
oops sorry! I have added it now